universal-login-react/src/core/utils/formatCurrency.ts

```diff
@@ -9,3 +9,8 @@ export const formatCurrency = (value: string, currency = 'USD'): string => {
   const formatter = new Intl.NumberFormat('en-US', {currency: 'USD', style});
   return formatter.format(withPrecision(Number.parseFloat(value || '0'), maximumFractionDigits));
 };
+
+export const formatUsdAmount = (amount: string, currency = 'USD') => {
```

Comment:
It is called formatUsdAmount but allows pass another currency. Maybe adjust formatCurrency to return formatted amount with tilde or 0

Reply:
What about formatFiatAmount?